### PR TITLE
Trigger image release on release being published

### PR DESCRIPTION
**Pull request by nebojsa-prodana**
We got releases working off manual tags in #107, now let us try to restore the functionality of #101 which should allow us to get a new tagged image once release drafter has published the release.

### CODE REVIEW

File: .github/workflows/release-artifacts.yaml [hunk 1]
```diff
@@ -34,5 +33,5 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: ${{ github.event.release.tag_name }}
```
Comment (nebojsa-prodana) [inline .github/workflows/release-artifacts.yaml:36]: See: https://stackoverflow.com/questions/59518658/how-to-use-github-release-version-number-in-github-action/59521156#59521156

Comment (DimitarHristov111) [reply .github/workflows/release-artifacts.yaml:36]: This way, we will not have the `latest` and `commit_sha` tags. Could this break something, somewhere? Just making sure that we have checked and we are sure 🙂

**Comment by sledigabel** [comment]
LGTM.

Re/ the trigger event, we use the same thing over in turbolift:
https://github.com/Skyscanner/turbolift/blob/main/.github/workflows/release.yml

**Review by sledigabel — APPROVED** [review]

File: .github/workflows/release-artifacts.yaml (hunk 1 above)
Comment (nebojsa-prodana) [reply .github/workflows/release-artifacts.yaml:36]: We don't have the latest anyways, so I think this is fine.

**Pull request merged**